mod-infra/src/main/scala/at/forsyte/apalache/infra/log/LogbackConfigurator.scala

```diff
@@ -24,21 +24,23 @@ class LogbackConfigurator(runDir: Option[Path], customRunDir: Option[Path]) exte
     setContext(loggerContext)
     runDir match {
       case Some(_) => configure(loggerContext)
-      case None    => configureSilent(loggerContext)
+      case None    => configureConsoleOnly(loggerContext)
     }
   }
 
-  def configureSilent(loggerContext: LoggerContext): Unit = {
+  def configureConsoleOnly(loggerContext: LoggerContext): Unit = {
```

Comment:
AFAIU, this only logs at and above `WARN`?

I would prefer to reflect this in the name, something like

```suggestion
  def configureConsoleOnlyWarn(loggerContext: LoggerContext): Unit = {
```